### PR TITLE
Add `meta` property for all rules

### PR DESCRIPTION
**Pull request by ybiquitous**
Each rule function should have properties like `ruleName`.
See https://stylelint.io/developer-guide/plugins

For example, `stylelint -f verbose` command can show rules' links:

image

### CODE REVIEW

File: src/utils/ruleUrl.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const revision = "master";
```
Comment (ybiquitous) [inline src/utils/ruleUrl.js:1]: [note] If using a tag (e.g. `v4.2.0`), the implementation would be more complex. I think the HEAD version (`master`) is enough.

**Comment by kristerkari** [comment]
Thanks! I'll try to get this reviewed ASAP.

File: src/utils/ruleUrl.js (hunk 1 above)
Comment (kristerkari) [reply src/utils/ruleUrl.js:1]: Using `master` sounds good enough.

**Review by kristerkari — APPROVED** [review]
Thanks! Looks good 👍

**Comment by ybiquitous** [comment]
@kristerkari Thank you for the review and merge! 😄

**Pull request merged**